packages/data-store/src/entities/validators/index.ts

```diff
@@ -0,0 +1,16 @@
+import { ValidationArguments, ValidatorConstraint, ValidatorConstraintInterface } from 'class-validator'
+
+@ValidatorConstraint({ name: 'isNonEmptyString', async: false })
+export class IsNonEmptyStringConstraint implements ValidatorConstraintInterface {
```

Comment:
This one is being used quite heavily. Given that a lot of the values actually are populated from external metadata sources, this could result in us throwing errors, simply because for instance an issuer had an empty string for an attribute. If we want to keep this everywhere, it means we will have to check in the mapping process from external data to the entity, whether a value is empty. If it is then store it as a null value. We should at least document that then

Reply:
Yes, that is a fair point. The reason I added this validation is to not have to deal with those values on the end, and a user can just check if the value is present. 

I have changed the code to make them undefined when saving them to the db